src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -75,6 +75,14 @@ cd "$__TOOLS_DIR/portableTargets"
 "$__DOTNET_CMD" restore --source https://www.myget.org/F/dotnet-buildtools/ --packages "$__TOOLS_DIR/portableTargets/packages/"
 cp -R "$__TOOLS_DIR/portableTargets/packages/Microsoft.Portable.Targets/${__PORTABLETARGETS_VERSION}/contentFiles/any/any/." "$__TOOLRUNTIME_DIR/."
 
+# Temporary workaround for CLI #710: copy native artifacts that weren't during dotnet publish.
+# This must be in sync with tool-runtime/project.json.
+__PACKAGE_DIR="$HOME/.nuget/packages"
+cp "$__PACKAGE_DIR/Microsoft.Build.Targets/0.1.0-preview-00017/runtimes/any/native/"* "$__TOOLRUNTIME_DIR"
+cp "$__PACKAGE_DIR/Microsoft.Net.Compilers.netcore/1.2.0-beta1-20160122-02/runtimes/any/native/"* "$__TOOLRUNTIME_DIR"
```

Comment:
Linux will be case-sensitive, and I believe the right path would be: Microsoft.Net.Compilers.**N**et**C**ore. That is what the project.lock.json says, but you should make sure that is correct.